server/src/main/java/eu/fasten/server/connectors/KafkaConnector.java

```diff
@@ -96,4 +97,10 @@ public static Properties kafkaProducerProperties(List<String> serverAddresses, S
                 "50000000"); //Set produce size to 50MB.
         return p;
     }
+
+    public static Properties cloneKafkaConsumerPropWithNewClientId(Properties consumerProperties, String suffix) {
+        Properties consumerPrioProperties = (Properties) SerializationUtils.clone(consumerProperties);
```

Comment:
This still uses a serialization call and a cast. Please get rid of that.

Reply:
Removed the method!